DiceKeys/Views/Scanning/CameraController.swift

```diff
@@ -70,7 +70,12 @@ final class DiceKeysCameraController: NSObject, AVCaptureVideoDataOutputSampleBu
             self.isProcessingFrame = false
             return
         }
+
+        #if os(macOS)
+        let ciImage = CIImage(cvPixelBuffer: imageBuffer).oriented(forExifOrientation: 8)
```

Comment:
Nice simple fix.  I had to look up that 8 == `CGImagePropertyOrientation.left`.

Better to use `CGImagePropertyOrientation.left` in place of `8`  and comment why the transform is needed.  A comment would be especially helpful since I actually don't know why this happens.  My best guess is...

```
// Frames are captured in portrait mode by default and need to be rotated 90 degrees counterclockwise on PCs
// where landscape is the default orientation of webcams
```

Do we know why?  Is that right?  If not, can you explain?

Do we know this code is correct on iOS when a device is flipped into landscape mode?